conans/client/generators/deploy.py

```diff
@@ -41,6 +42,13 @@ def content(self):
                                        os.path.relpath(root, rootpath), f)
                     dst = os.path.normpath(dst)
                     mkdir(os.path.dirname(dst))
-                    shutil.copy(src, dst)
+                    if six.PY2:
+                        if os.path.islink(src):
+                            linkto = os.readlink(src)
+                            os.symlink(linkto, dst)
```

Comment:
This might not be accurate. When copying things from the cache to the user space, you expect to copy the relative link to it, but not to point to the cache file.

Reply:
Fixed